app/src/test/java/com/codesoom/assignment/application/UserCommandServiceTest.java

```diff
@@ -0,0 +1,93 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.Fixture;
+import com.codesoom.assignment.domain.Role;
+import com.codesoom.assignment.domain.UserRepository;
+import com.codesoom.assignment.dto.UserInquiryInfo;
+import com.codesoom.assignment.errors.DuplicatedEmailException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+@SpringBootTest
+@DisplayName("UserService 인터페이스의")
```

Comment:
인터페이스라는 것을 굳이 명시해주지 않아도 됩니다. 인터페이스라는 건 참고사항일 뿐이죠. 만약 `UserService`를 클래스로 선언했다 하더라도 이 테스트의 내용은 모두 참이어야 합니다.

```suggestion
@DisplayName("UserService의")
```